input/resources/NamingSystem-Danish-provider-registry---Ydernummer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "resourceType": "NamingSystem",
+    "id": "ydernummer",
+    "name": "Ydernummer",
+    "status": "active",
+    "kind": "identifier",
+    "date": "1968-04-02T00:00:00+00:00",
+    "publisher": "Sundhedsdatastyrelsen", 
+    "contact": [
+        {
+            "name": "Mikkel Wermer Steen"
+        }
+    ],
+    "responsible": "Sundhedsdatastyrelsen",
```

Comment:
From https://www.esundhed.dk/Dokumentation?rid=10&tid=47&vid=272 and https://sundhedsdatastyrelsen.dk/data-og-registre/nationale-sundhedsregistre/yderregisteret